qupath-gui-fx/src/main/java/qupath/lib/gui/commands/PathObjectGridView.java

```diff
@@ -584,24 +571,22 @@ private void updateChildren() {
 			}
 			List<Node> images = new ArrayList<>();
 			for (PathObject pathObject : list) {
-				Label viewNode = nodeMap.get(pathObject);
-				if (viewNode == null) {
-					
+				Label viewNode = nodeMap.computeIfAbsent(pathObject, po -> {
```

Comment:
Admittedly the starting code wasn't great and the switch to `computeIfAbsent` makes sense, but could this be simplified?

It's quite hard to read the long lambda, and then there is another lambda nested inside to handle the mouse clicks. Is there any reason this can't be split out into its own method?